src/index .html

```diff
@@ -0,0 +1,196 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      >
+      <fieldset class="form">
+        <legend>Peronal infromation</legend>
+        <div class="form-field">
+          <label for="form-field">
+            Surname:
+            <input
+              type="text"
+              name="surname"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="form-field">
+            Name:
+            <input
+              type="text"
+              name="name"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="form-field">
+            How old are You?
+            <input
+              type="number"
+              name="age"
+              value="12"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="form-field">
+            Full date of birth:
+            <input
+              type="date"
+              name="dateOfBirth"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="form-field">
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="accept"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="form">
+        <legend>Registration</legend>
+
+        <div class="form-field">
+          <label for="form-field">
+            E-mail:
+            <input
+              type="email"
+              name="email"
+              placeholder="email@example.com"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="form-field">
+            Password:
+            <input
+              type="password"
+              name="password"
+              required
+              minlength="8"
+              maxlength="50"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="form">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          <label for="form-field">
+            Do you love cats
+            <input
+              type="radio"
+              name="cats"
+            >Yes
+
+            <input
+              type="radio"
+              name="cats"
+            >No
+          </label>
```

Comment:
not great style
- `Yes` and `No` should be indented on same level as the `<input`
- you're missing value attribute for each of inputs
- would be nice to label `Yes` and `No` as well to increase a clickable area of each option
```suggestion
          <label for="form-field">
            Do you love cats
            <label>
              <input
                type="radio"
                name="cats"
                ?value?
              >
              Yes
            </label>
            
            <label>
              <input
                type="radio"
                name="cats"
                ?value?
              >
              No
            <label>
          </label>
```